plugins/org.jkiss.dbeaver.ext.gaussdb.ui/src/org/jkiss/dbeaver/ext/gaussdb/ui/views/GaussDBConnectionPage.java

```diff
@@ -215,12 +219,16 @@ public void loadSettings() {
         setupConnectionModeSelection(urlText, useURL, GROUP_CONNECTION_ARR);
         updateUrl();
 
-        activated = false;
+        activated = true;
     }
 
     @Override
     public void saveSettings(DBPDataSourceContainer dataSource) {
         DBPConnectionConfiguration connectionInfo = dataSource.getConnectionConfiguration();
+        if (typeURLRadio != null) {
+            connectionInfo.setConfigurationType(
+                    typeURLRadio.getSelection() ? DBPDriverConfigurationType.URL : DBPDriverConfigurationType.MANUAL);
```

Comment:
We use four spaces for continuous indentation

```suggestion
                typeURLRadio.getSelection() ? DBPDriverConfigurationType.URL : DBPDriverConfigurationType.MANUAL);
```

Reply:
Thank you so much for your suggestion. The problem has been solved.